### DISCUSSION TITLE
Remove other node versions?

### DISCUSSION THREAD

**Discussion by mattbell87** [2pisoftware/cmfive-core#383] [Polls]
Currently we have a matrix that tests against other node versions. We use node for two reasons:

- Playwright
- Building the theme

In other words we do not use node in production, and node isnt required to run Cosine. Its only used for compiling and testing.

Since we have a dockerfile that compiles the theme, as well as a container for live development, and a CodePipeline that also compiles it, do we need this matrix and do we need to support other node versions at all? Or can we support 1 node version and cut down build minutes?

**Comment by chris-bateman** [comment]
Based on this - https://endoflife.date/nodejs
Let's support 2x LTS versions.  The latest LTS and the one before.  This ensures we progress regularly towards the next LTS and have a fallback option.

So 20 + 22 for example.  Then, when 24 comes out, we drop 20 and move any remaining systems to 22 and leave room for troubleshooting any issues on 24 as they arise.
